### ISSUE TITLE
Unable to load pubsubs from server and items from pubsub group

### ISSUE THREAD

**Issue by pztrn** [movim/movim#394]
While trying to access, for example, pubsub.feder8.ru, Movim shows nothing and this appears in JS console:

```
Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at WebSocket.connection.onmessage
```

Checked from my instance and from nl.movim.eu with different accounts (my server and jabber.ru).

Here what I have in console from my movim instance:

```
May 27 17:06:17 web modl[12225]: modl.INFO: insert into item (server, creator, node, jid, name, created, updated, description, logo ) values( :server, :creator, :node, :jid, :name, :created, :updated, :description, :logo ) {"name":"feder8-news","creator":null,"created":null,"updated":"2017-05-27 14:06:17","server":"pubsub.feder8.ru","jid":"pubsub.feder8.ru","node":"feder8-news","description":null,"logo":false} []
May 27 17:06:17 web modl[12225]: modl.INFO: s:268:"7 : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "item_prim_key" DETAIL:  Ключ "(server, jid, node)=(pubsub.feder8.ru, pubsub.feder8.ru, feder8)" уже существует."; [] []
May 27 17:06:17 web modl[12225]: modl.INFO: insert into item (server, creator, node, jid, name, created, updated, description, logo ) values( :server, :creator, :node, :jid, :name, :created, :updated, :description, :logo ) {"name":"feder8","creator":null,"created":null,"updated":"2017-05-27 14:06:17","server":"pubsub.feder8.ru","jid":"pubsub.feder8.ru","node":"feder8","description":null,"logo":false} []
May 27 17:06:17 web modl[12225]: modl.INFO: s:275:"7 : ОШИБКА:  повторяющееся значение ключа нарушает ограничение уникальности "item_prim_key" DETAIL:  Ключ "(server, jid, node)=(pubsub.feder8.ru, pubsub.feder8.ru, russian-movim)" уже существует."; [] []
May 27 17:06:17 web modl[12225]: modl.INFO: insert into item (server, creator, node, jid, name, created, updated, description, logo ) values( :server, :creator, :node, :jid, :name, :created, :updated, :description, :logo ) {"name":"russian-movim","creator":null,"created":null,"updated":"2017-05-27 14:06:17","server":"pubsub.feder8.ru","jid":"pubsub.feder8.ru","node":"russian-movim","description":null,"logo":false} []
```

It says about unique keys errors that already exists. (sorry, my pgsql inherited ru_RU from environment).

I'm on Gentoo with latest php5 and pgsql 9.5.1. Ejabberd 17.03-beta as XMPP server.

**Comment by pztrn** [comment]
Same on updated Debian Jessie. That's what I learned:

If I tried to access **remote** pubsub server for first time - everything is okay and I can enter it **only for the very first time**. For next times it shows only loading screen. Appears on newly installed movim. I mean this:

* Account on example.com can explore pubsub.foobar.net only once, all other times example.com will get loading screen with errors in first post.
* Account on example.com can explore pubsub.example.com without any problem.
